cli/src/index.ts

```diff
@@ -31,7 +31,9 @@ const main = async () => {
   const npmVersion = await getNpmVersion();
   const pkgManager = getUserPkgManager();
   renderTitle();
-  npmVersion && renderVersionWarning(npmVersion);
+  if (npmVersion) {
+    renderVersionWarning(npmVersion);
+  }
```

Comment:
[`@typescript-eslint/no-unused-expressions`](https://typescript-eslint.io/rules/no-unused-expressions) was promoted to the [`recommended` preset](https://v8--typescript-eslint.netlify.app/users/configs#recommended) in v8.